internal/core/analysis_api/handlers/list.go

```diff
@@ -74,7 +74,7 @@ var (
 // like lastModified.
 
 // Dynamo filters common to both ListPolicies and ListRules
-func pythonListFilters(enabled *bool, nameContains, severity string, types, tags []string) []expression.ConditionBuilder {
+func pythonListFilters(enabled *bool, nameContains string, severity []compliancemodels.Severity, types, tags []string) []expression.ConditionBuilder {
```

Comment:
This line is now too long (which is why `test:go` failed in CI):

```suggestion
func pythonListFilters(
    enabled *bool,
    nameContains string,
    severity []compliancemodels.Severity,
    types, tags []string,
) []expression.ConditionBuilder {
```